amx/server/natives/a_samp.lua

```diff
@@ -19,13 +18,13 @@ skinReplace = {
 	[273] = 0,
 }
 --replace colors
-function colorizeString(string) 
+function colorizeString(string)
 	return string:gsub("(=?{[0-9A-Fa-f]*})",
-	function(colorMatches)
-		colorMatches = colorMatches:gsub("[{}]+", "") --replace the curly brackets with nothing
-		colorMatches = '#' .. colorMatches --Append to the beginning
-		return colorMatches 
-	end)
+		function(colorMatches)
```

Comment:
What are all these CRLF changes :'(

Reply:
I think you should break down this PR into chunks as in make multiple PRs and organize them correctly, explain what it is that you changed. I saw that you added vehicle spawn txts, that should be in another PR.

There's also a lot of unneeded carriage difference stuff that just clutters the PR entirely :/

It is very difficult and time consuming for me to go through all these changes. This PR *changes a lot of stuff*. It's also been two years since I last committed to this repo.
How do I know this won't break compatibility with anything?

Thanks for the effort though! :-)

Reply:
> What are all these CR/LF changes? :'(

my shitty .editorconfig broken all the files

Reply:
i tried **core.autocrlf=input** & **core.eol=lf** & **core.safecrlf=false** and 
```
git rm --cached -r .
git reset --hard
git add --renormalize .
```

but it seems like these PRs are not worth merging.... and I will focus on managing my fork since it is a hopeless effort-wasting and time-consuming pull-requests (while merging any change like PR or commit into my fork)... I felt off trying to split commits and fixing line endings -_-